selfdrive/boardd/panda.cc

```diff
@@ -338,6 +338,10 @@ void Panda::set_power_saving(bool power_saving) {
   usb_write(0xe7, power_saving, 0);
 }
 
+void Panda::set_deepsleep(void) {
```

Comment:
I'd call this enable_deepsleep. set sounds like it takes an arg to enable/disable